person.rb

```diff
@@ -25,4 +27,8 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(book, date)
+    @rentals.push(Rental.new(date, book, self))
+  end
```

Comment:
- [ ] Kindly make this function only create a new rental as you already add it to the rentals list in the Rental class initialize method. This line will make the same rental added twice to the list so that it will be like this:
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
@gemmen29 Thanks fro your suggestion. Corrected the same.